tests/build/build.go

```diff
@@ -348,31 +349,34 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				Expect(ok).To(BeTrue())
 				Expect(expiration).To(Equal(utils.GetEnv(constants.IMAGE_TAG_EXPIRATION_ENV, constants.DefaultImageTagExpiration)))
 			})
-			It("eventually leads to a creation of a PR comment with the PipelineRun status report", func() {
-				var comments []*github.IssueComment
+			It("eventually leads to the PipelineRun status report at Checks tab", func() {
```

Comment:
since we're moving away from PaC Webhooks to GitHub App, should we also update the rhtap-demo test?

Reply:
@psturc could you let me know which ones of the rhtap-demo test should be updated?

Reply:
https://github.com/redhat-appstudio/e2e-tests/blob/170867a2b85dd1a0c518073728c54aba370d71c3/tests/rhtap-demo/rhtap-demo.go#L408

Reply:
there is no error, no need to be updated.